recipes/r-mlapi/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = '0.1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-mlapi
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: mlapi_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/mlapi_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/mlapi/mlapi_{{ version }}.tar.gz
+  sha256: 3314e96f4a467a9b6337b4fb16bc5a22e5dee641745ea511c24c327457997b08
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-matrix >=1.1
+    - r-r6 >=2.2.1
+
+  run:
+    - r-base
+    - r-matrix >=1.1
+    - r-r6 >=2.2.1
+
+test:
+  commands:
+    - $R -e "library('mlapi')"  # [not win]
+    - "\"%R%\" -e \"library('mlapi')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=mlapi
+  license: MIT
+  summary: Provides 'R6' abstract classes for building machine learning models  with 'scikit-learn'
```

Comment:
The summary is truncated; that usually happens when the skeleton helper encounters a ':'...

Reply:
I fixed the summary.